src/Stripe.net/Constants/StripeSourceType.cs

```diff
@@ -2,6 +2,7 @@
 {
     public static class StripeSourceType
     {
+        public const string ACHCreditTransfer = "ach_credit_transfer";
```

Comment:
Sorry for the nit @ob-stripe, but I think the convention in this library is the "lowercase acronym style" as seen by property names like `Id` versus `ID`. Mind if we call this `AchCreditTransfer` instead?